src/DynamoUtilities/Md2Html.cs

```diff
@@ -0,0 +1,173 @@
+        {
+            KillProcess();
+        }
+
+        /// <summary>
+        /// Kill the CLI tool, if still running
+        /// </summary>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        /// <summary>
+        /// Converts a markdown string into Html.
+        /// </summary>
+        /// <param name="mdString"></param>
+        /// <param name="mdPath"></param>
+        /// <returns>Returns converted markdown as html</returns>
+        internal string ParseMd2Html(string mdString, string mdPath)
+        {
+            if (!started)
+            {
+                return GetErrorMessage();
+            }
+
+            process.StandardInput.WriteLine(@"<<<<<Convert>>>>>");
+            process.StandardInput.WriteLine(mdPath);
+            process.StandardInput.WriteLine(mdString);
+            process.StandardInput.WriteLine(@"<<<<<Eod>>>>>");
+
+            var writer = new StringWriter();
+            GetData(ref writer);
+
+            return writer.ToString();
+        }
+
+        /// <summary>
+        /// Sanitize Html
+        /// </summary>
+        /// <param name="content"></param>
+        /// <returns>Returns Sanitized Html</returns>
+        internal string SanitizeHtml(string content)
+        {
+            if (!started)
+            {
+                return GetErrorMessage();
+            }
+
+            process.StandardInput.WriteLine(@"<<<<<Sanitize>>>>>");
+            process.StandardInput.WriteLine(content);
+            process.StandardInput.WriteLine(@"<<<<<Eod>>>>>");
+
+            var writer = new StringWriter();
+            GetData(ref writer);
+
+            return writer.ToString();
+        }
+
+        /// <summary>
+        /// Read data from CLI tool
+        /// <param name="writer"></param>
+        /// </summary>
+        private void GetData(ref StringWriter writer)
+        {
+            var done = false;
+
+            while (!done)
+            {
+                var line = process.StandardOutput.ReadLine();
+                if (line == null || line == @"<<<<<Eod>>>>>")
+                {
+                    done = true;
+                }
+                else
+                {
+                    if (!string.IsNullOrWhiteSpace(line))
+                    {
+                        writer.WriteLine(line);
+                    }
+                }
+            }
+        }
+
+
+        /// <summary>
+        /// Compute the location of the CLI tool
+        /// </summary>
+        /// <returns>Returns full path to the CLI tool</returns>
+        private static string GetToolPath ()
+        {
+            var rootPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location) ?? throw new ArgumentNullException("Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)");
+            var toolPath = Path.Combine(rootPath, @"Md2Html\Md2Html.exe");
```

Comment:
I don't know the answer to this - does `Path.Combine` generate a valid path cross platform even if you use hardcoded directory separator characters?

Reply:
Good point. Let me fix that.